eng/native/build-commons.sh

```diff
@@ -272,7 +272,11 @@ elif [[ "$platform" == "NetBSD" || "$platform" == "SunOS" ]]; then
 elif [[ "$platform" == "Darwin" ]]; then
   __NumProc=$(($(getconf _NPROCESSORS_ONLN)+1))
 else
-  __NumProc=$(nproc --all)
+  if which nproc > /dev/null ; then
```

Comment:
Can we use command instead, to improve portability?

```suggestion
  if command -v nproc > /dev/null 2>&1; then
```

Reply:
Yes, that's better, thanks!